aws/resource_aws_ec2_transit_gateway_peering_attachment_accepter.go

```diff
@@ -0,0 +1,205 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceAwsEc2TransitGatewayPeeringAttachmentAccepter() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEc2TransitGatewayPeeringAttachmentAccepterCreate,
+		Read:   resourceAwsEc2TransitGatewayPeeringAttachmentAccepterRead,
+		Update: resourceAwsEc2TransitGatewayPeeringAttachmentAccepterUpdate,
+		Delete: resourceAwsEc2TransitGatewayPeeringAttachmentAccepterDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"peer_account_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"peer_region": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"peer_transit_gateway_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"tags": tagsSchema(),
+			"transit_gateway_attachment_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"transit_gateway_default_route_table_association": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"transit_gateway_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceAwsEc2TransitGatewayPeeringAttachmentAccepterCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).ec2conn
+
+	input := &ec2.AcceptTransitGatewayPeeringAttachmentInput{
+		TransitGatewayAttachmentId: aws.String(d.Get("transit_gateway_attachment_id").(string)),
+	}
+
+	log.Printf("[DEBUG] Accepting EC2 Transit Gateway Peering Attachment: %s", input)
+	output, err := conn.AcceptTransitGatewayPeeringAttachment(input)
+	if err != nil {
+		return fmt.Errorf("error accepting EC2 Transit Gateway Peering Attachment: %s", err)
+	}
+
+	d.SetId(aws.StringValue(output.TransitGatewayPeeringAttachment.TransitGatewayAttachmentId))
+	transitGatewayID := aws.StringValue(output.TransitGatewayPeeringAttachment.AccepterTgwInfo.TransitGatewayId)
+
+	if err := waitForEc2TransitGatewayPeeringAttachmentAcceptance(conn, d.Id()); err != nil {
+		return fmt.Errorf("error waiting for EC2 Transit Gateway Peering Attachment (%s) availability: %s", d.Id(), err)
+	}
+
+	if err := setTags(conn, d); err != nil {
+		return fmt.Errorf("error updating EC2 Transit Gateway Peering Attachment (%s) tags: %s", d.Id(), err)
+	}
```

Comment:
Tagging is now handled by the shared `keyvaluetags` package, please see the [Adding Resource Tagging Support section of the Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#adding-resource-tagging-support) for more information.

```suggestion
	if v := d.Get("tags").(map[string]interface{}); len(v) > 0 {
		if err := keyvaluetags.Ec2CreateTags(conn, d.Id(), v); err != nil {
			return fmt.Errorf("error updating EC2 Transit Gateway Peering Attachment (%s) tags: %s", d.Id(), err)
		}
	}
```